logback-contrib/pom.xml

```diff
@@ -120,136 +107,33 @@
     </dependencies>
 
     <dependencyManagement>
-
         <dependencies>
 
-
             <!-- Pool of dependencies available to any sub-module, but not included automatically.
                 They must be explicitly referenced in the children POMs, but _don't_ include
                 version numbers in the child POMs.  Define them here only to guarantee version
                 compatibility across children modules! -->
 
+            <dependency>
+                <groupId>org.apache.james</groupId>
+                <artifactId>james-project</artifactId>
+                <version>3.9.0-SNAPSHOT</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+
             <!-- Logback Extensions sub-modules dependency .jars (alphabetized based on artifactId for easy scanning please!): -->
-          <dependency>
-               <groupId>ch.qos.logback</groupId>
-               <artifactId>logback-access</artifactId>
-               <version>${logback.version}</version>
-           </dependency>
-
-          <dependency>
-               <groupId>ch.qos.logback</groupId>
-               <artifactId>logback-classic</artifactId>
-               <version>${logback.version}</version>
-           </dependency>
-
-          <dependency>
-               <groupId>ch.qos.logback</groupId>
-               <artifactId>logback-core</artifactId>
-               <version>${logback.version}</version>
-           </dependency>
+            <dependency>
+                <groupId>ch.qos.logback</groupId>
+                <artifactId>logback-access</artifactId>
+                <version>${logback.version}</version>
+            </dependency>
 
             <dependency>
-                <groupId>com.fasterxml.jackson.core</groupId>
-                <artifactId>jackson-databind</artifactId>
-                <version>${jackson.version}</version>
-                <scope>provided</scope>
+                <groupId>ch.qos.logback</groupId>
+                <artifactId>logback-core</artifactId>
+                <version>${logback.version}</version>
```

Comment:
We shall be able to remove the logback-core from the dependency management.

Reply:
i could remove it from the dependency management and specify the version directly in `logback-contrib/json/core/pom.xml` ?
we will have to specify it at some point because it not defined in any parent pom, nor in james-project.

Reply:
Oh then its ok, but we must ensure logback versions are aligned...